client/src/app/components/manage-organizations/organization-view/organization-view.component.ts

```diff
@@ -36,7 +41,58 @@ export abstract class OrganizationViewComponent implements OnInit {
     }
   }
 
+  populateExcelElem(): void {
+    for (const elem of this.linesR) {
+      this.locations.push(elem);
+    }
+  }
+
+  populateUserInputElem(): void {
+    const locArr = this.location.split(',');
+    // tslint:disable-next-line:prefer-for-of
+    for (let i = 0; i < locArr.length; i += 2) {
+      if (i !== locArr.length - 1) {
+        const tempArr = [];
+        tempArr.push(locArr[i]);
+        tempArr.push(locArr[i + 1]);
+        // @ts-ignore
+        this.locations.push(tempArr);
+      }
+    }
+  }
+
+  // tslint:disable-next-line:typedef
+  public changeListener(files: FileList) {
```

Comment:
Could you be more specific here with the method name ? Or add comments as to what exactly this is doing.